### DISCUSSION TITLE
Access existing Flux (logs)

### DISCUSSION THREAD

**Discussion by btzs** [fluxcd/flux2#2687] [Q&A] [answered]
Hi!
I am new to Flux. 
I have cluster A and cluster B. Cluster A is my test cluster and I set up Flux from the beginning following the tutorial. That worked. Cluster B is an already existing cluster. This does not reconcile successfully, so the cluster is not updated. 

I want to see the reconciliation log of cluster B.

```
$ flux version --namespace devops

flux: v0.27.3
helm-controller: v0.11.2
kustomize-controller: v0.14.0
notification-controller: v0.16.0
source-controller: v0.15.4
```
Flux is installed in namespace devops:
```
$ kubectl get pods --namespace devops

NAME                                       READY   STATUS    RESTARTS      AGE
helm-controller-5c4c9b8fdb-8w85x           1/1     Running   0             29h
kustomize-controller-5fb9cdbd44-s9tpf      1/1     Running   0             29h
notification-controller-5846cf794f-ncl6b   1/1     Running   0             29h
source-controller-5755566899-768jm         1/1     Running   1 (14h ago)   29h
```

The `error scheme "ssh" not supported` is what I actually want to find out more about.
```
$ flux  get  all --namespace devops
NAME                    READY   MESSAGE                                 REVISION        SUSPENDED 
gitrepository/devops    True    Fetched revision: master/99e6asd        master/99e6asd  False    

NAME                            READY   MESSAGE                         REVISION        SUSPENDED 
helmrepository/myapp-helm-uat False   scheme "ssh" not supported                      False  
```  

```
$ flux logs --all-namespaces
$
```

I don't understand way the last command doesn't display anything. Do I have to change anything?

**Comment by souleb** [comment, marked as answer]
did you try `flux logs --flux-namespace=my-namespace`?

**Comment by stefanprodan** [comment]
The default namespace for all Flux CLI operations is `flux-system`. If you choses to use some other namespace when install Flux, then you have to tell the CLI which namespace hosts the Flux controllers by setting this env var:

```
export FLUX_SYSTEM_NAMESPACE=devops
```

**Comment by btzs** [comment]
Thanks for your help!
`flux logs --flux-namespace=my-namespace` worked!

@stefanprodan:
Setting the environment variable didn't work. I saw that this was only added in November '21 ([your commit](https://github.com/fluxcd/flux2/commit/f2de7e04b8e6f3a8f98dcfad02ff57c1cdf6acb9)). Maybe my flux is too old? Thanks anyways!
